src/lib.rs

```diff
@@ -27,6 +27,9 @@
 //! groups. Groups can be created, updated, and deleted along with management of a groups administrators and members.
 //!
 
+// required by quick_error or IronOxideErr
+#![recursion_limit = "128"]
```

Comment:
:confused: Why?

Reply:
quick_error macro must process the errors recursively? All I know is that I added an error and I broke the standard limit of 64/

Reply:
huh, ok.